lib/IRGen/LoadableByAddress.cpp

```diff
@@ -1426,6 +1426,22 @@ static void convertBBArgType(SILBuilder &argBuilder, SILType newSILType,
   }
 }
 
+static bool containsFunctionType(SILType ty) {
```

Comment:
I think it would be simpler if this were an operation on Type, and then the caller can pass in resultStorageTy.getSwiftType()